app/models/manageiq/providers/atomic/container_manager.rb

```diff
@@ -13,7 +13,7 @@ def self.ems_type
   end
 
   def self.description
-    @description ||= "Atomic".freeze
+    @description ||= "Atomic Project".freeze
```

Comment:
@chessbyte we have an icon that is based on that shape but it was designed by @serenamarie125 for ManageIQ.